### PR TITLE
Add alias for commands

### PR DESCRIPTION
**Pull request by ktnc985**
### Add a shorthand alias for the following commands:
1. "a" for Add command
2. "c" for Clear command
3. "d" for Delete command
4. "e" for Edit command
5. "f" for Find command
6. "l" for List command
7. "r" for Redo command
8. "s" for Select command
9. "u" for Undo command

### CODE REVIEW

**Comment by Kennard123661** [comment]
Should we add alias? would it make our commands more abstract? oO

**Comment by Kennard123661** [comment]
also, what if we introduce more commands, then it will become kinda confusing, like design wise is this a good idea? @Kowalski985

**Comment by ktnc985** [comment]
I think it will make our application more flexible for the end user. If we're moving towards a more chatbot like interface, our commands won't have a very concrete format to them and should be able to handle a range of formats, though this would be challenging on our end.

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -49,24 +49,31 @@ public Command parseCommand(String userInput) throws ParseException {
...
         case AddCommand.COMMAND_WORD:
+        case AddCommand.COMMAND_ALIAS:
             return new AddCommandParser().parse(arguments);
 
         case EditCommand.COMMAND_WORD:
+        case EditCommand.COMMAND_ALIAS:
             return new EditCommandParser().parse(arguments);
 
         case SelectCommand.COMMAND_WORD:
+        case SelectCommand.COMMAND_ALIAS:
```
Comment (Kennard123661) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java]: add fall through comment for all @Kowalski985

**Review by Kennard123661 — COMMENTED** [review]
Update the user + developer guide and update the examples usage. @Kowalski985

**Pull request closed without merging**